float64 timestamp

# Vicon data
float64 vicon_x
float64 vicon_y
float64 vicon_z
float64 vicon_roll
float64 vicon_pitch
float64 vicon_yaw

# Camera data
float64 camera_x
float64 camera_y
float64 camera_z
float64 camera_roll
float64 camera_pitch
float64 camera_yaw

